### PR TITLE
don't add packaging as codeowners for test.rb and development.rb

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Gemfile         @theforeman/packaging
-Rakefile.dist	@theforeman/packaging
-bundler.d/*.rb  @theforeman/packaging
+Rakefile.dist   @theforeman/packaging
 package.json    @theforeman/packaging
+bundler.d/      @theforeman/packaging
+bundler.d/test.rb
+bundler.d/development.rb
```
Comment (ekohl) [inline .github/CODEOWNERS]: It was sorted

**Review by tbrisker — DISMISSED** [review]
I'm not sure about the syntax but agree with the general idea

**Comment by tbrisker** [comment]
also, aborted CI pipelines. We should probably add some way to prevent them from auto-triggering on PRs.

File: .github/CODEOWNERS (hunk 1 above)
Comment (evgeni) [reply .github/CODEOWNERS]: "sorta", I found it easier to read with the exceptions at the very end.

can re-sort if you like.

Comment (evgeni) [reply .github/CODEOWNERS]: that'd be

```
Gemfile         @theforeman/packaging
Rakefile.dist   @theforeman/packaging
bundler.d/      @theforeman/packaging
bundler.d/development.rb
bundler.d/test.rb
package.json    @theforeman/packaging
```

**Comment by evgeni** [comment]
Syntax is based on what I read in https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

aka:
- packging owns the bundler.d folder, but then the two files have explicitly no owners.

**Review by ekohl — DISMISSED** [review]
You can also add sections by separating them by whitespace. Perhaps even describe why we exclude those files.

**Comment by evgeni** [comment]
@ekohl like this?

**Review by ekohl — APPROVED** [review]
Thanks!

**Pull request merged**